src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,29 @@
 import React from 'react';
+import { Comment } from '../../types/comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment,
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
+  <>
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{comment.name}</strong>
```

Comment:
Let's destructure the comment 
`const { name, email, body } = comment`

Reply:
done